docs/sources/developer/Dependencies.md

```diff
@@ -1,48 +1,36 @@
 # Plaso Dependencies
 
-This page contains detailed instructions on how to build and install dependencies.
+This page contains detailed instructions on how to install dependencies for 
+development.
 
 There are multiple ways to install the dependencies:
 
-* Using [prepackaged dependencies](Dependencies.md#prepackaged-dependencies);
-* [Batch build](Dependencies.md#batch-build) most of the dependencies;
-* [Manual build](Dependencies.md#manual-build) of the dependencies.
+* [Using pre-packaged dependencies](Dependencies.md#prepackaged-dependencies)
+* [Batch build](Dependencies.md#batch-build)
+* [Manual build](Dependencies.md#manual-build)
 
 ## Optional dependencies
 
-Some dependencies for plaso are optional:
+Some dependencies for Plaso are optional:
 
-* Timesketch: https://github.com/google/timesketch/blob/master/docs/Installation.md
+* [Timesketch](https://github.com/google/timesketch/blob/master/docs/Installation.md) 
 
 ## Prepackaged dependencies
 
 ### Fedora Core
 
-**Note that the instructions in this page assume you are running on Fedora Core 26, 27 or 28. Installing packages from the copr on other versions and/or distributions is not recommended.**
+**Note that these instructions assume you are running on Fedora Core 28, 29 
```

Comment:
remove Core in Fedora Core, also see comment about https://github.com/log2timeline/plaso/pull/2664